connect/api/src/main/java/org/apache/kafka/connect/health/ConnectClusterState.java

```diff
@@ -17,29 +17,29 @@
 
 package org.apache.kafka.connect.health;
 
-import org.apache.kafka.connect.rest.ConnectRestExtension;
-
 import java.util.Collection;
 
 /**
- * Provides the ability to lookup connector metadata and its health. This is made available to
- * the {@link ConnectRestExtension} implementations. Connect Framework provides the implementation
- * for this interface.
+ * Provides the ability to lookup connector metadata and its health. This is made available to the {@link org.apache.kafka.connect.rest.ConnectRestExtension}
+ * implementations. Connect Framework provides the implementation for this interface.
```

Comment:
"The Connect framework", not "Connect Framework". Check this elsewhere, since it seems like I've requested this change multiple times already and the usage in this commit still is not consistent. :-)